frontend/src/shared/modules/filters/services/filter-query.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Filter, FilterConfig } from '@/shared/modules/filters/types/FilterConfig';
+import { queryUrlParserByType } from '@/shared/modules/filters/config/queryUrlParserByType';
+import { CustomFilterConfig } from '@/shared/modules/filters/types/filterTypes/CustomFilterConfig';
+
+export const filterQueryService = () => {
```

Comment:
Could you add some comments on this file?
I think it will be simpler to understand them if we have some descriptions